.travis.yml

```diff
@@ -43,7 +43,7 @@ install:
   - echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
 
 script:
-  - TERM=dumb tools/test --full
+  - TERM=dumb travis_wait tools/test --full
```

Comment:
Interesting -- what's this bit do?

Reply:
This is used to ensure that this command keeps on running for 20 minutes even if it doesn't print any output. More information can be found here https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received